### PR TITLE
add type of elements in list in package modules

### PR DESCRIPTION
**Pull request by nikita-b**
##### SUMMARY
Add type of elements in list in package module for better error handling

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
apt, package
lib/ansible/modules/packaging/

##### ADDITIONAL INFORMATION

### CODE REVIEW

File: lib/ansible/modules/packaging/os/portage.py [hunk 1]
```diff
@@ -462,7 +462,7 @@ def run_emerge(module, packages, *args):
 def main():
     module = AnsibleModule(
         argument_spec=dict(
-            package=dict(default=None, aliases=['name'], type='list'),
+            package=dict(type='list', default=None, aliases=['name']),
```
Comment (tchernomax) [inline lib/ansible/modules/packaging/os/portage.py]: You didn't add elements='str' here, was it intended ?

**Review by tchernomax — COMMENTED** [review]
Was it intended you didn't update the documentation of portage ?
```
type: list
elements: srt
```

File: lib/ansible/modules/packaging/os/portage.py (hunk 1 above)
Comment (nikita-b) [reply lib/ansible/modules/packaging/os/portage.py]: Yep! I didn't have apportunity to test on Gentoo and just fix style :)

Comment (wltjr) [reply lib/ansible/modules/packaging/os/portage.py]: Sorry, I did not realize this affected the portage module. I can see about testing if others do not have time. Offhand, I do not see it causing any issues.

Comment (nikita-b) [reply lib/ansible/modules/packaging/os/portage.py]: You are right. It's just my first contribution and I'm little cowardly. I will check if this change is needed for other modules.

Comment (nikita-b) [reply lib/ansible/modules/packaging/os/portage.py]: I add option for portage and homebrew\macports.

**Review by tchernomax — APPROVED** [review]
LGTM

**Comment by acozine** [comment]
Thanks @nikita-b for helping to make the Ansible documentation better!

**Comment by acozine** [comment]
rebuild_merge

**Pull request merged**